source/examples/comparison/glbinding.cpp

```diff
@@ -45,3 +47,15 @@ void glbinding_error(bool enable)
     else
         glbinding::setCallbackMask(glbinding::CallbackMask::None);
 }
+
+void glbinding_log(bool enable)
```

Comment:
I suggest you provide a convenience method on ```Logging``` that takes the boolean parameter and dispatches to ```start()``` and ```stop()``` internally.